src/main/java/com/epam/dataservice/accident/AccidentWriter.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.dataservice.accident;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.concurrent.BlockingQueue;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+import com.epam.data.RoadAccident;
+
+public class AccidentWriter implements Runnable {
+
+	private BlockingQueue<RoadAccident> accidentQueue;
+	private String accidentFile;
+
+	public AccidentWriter(BlockingQueue<RoadAccident> accidentQueue, String accidentFile) {
+		this.accidentQueue = accidentQueue;
+		this.accidentFile = accidentFile;
+	}
+
+	@Override
+	public void run() {
+		CSVPrinter csvPrinter = null;
+		try {
+			FileWriter accidentFileWriter = new FileWriter(this.accidentFile);
+			csvPrinter = new CSVPrinter(accidentFileWriter, CSVFormat.DEFAULT.withRecordSeparator("\n"));
+			csvPrinter.printRecord("Accident_Index", "Longitude", "Latitude", "Accident_Hour");
+			while (true) {
+				RoadAccident accident = accidentQueue.take();
+				csvPrinter.printRecord(accident.getAccidentId(), accident.getLongitude(), accident.getLatitude(), accident.getTime().getHour());
+			}
+		} catch (InterruptedException e) {
```

Comment:
North, never live empty Exception. At least leave comment why it's empty.  It's extremely tough to find error in that case